flutter/lib/src/default_integrations.dart

```diff
@@ -358,15 +358,6 @@ class LoadReleaseIntegration extends Integration<SentryFlutterOptions> {
 
         options.release = release;
         options.dist = packageInfo.buildNumber;
```

Comment:
```suggestion
        options.release = options.release ?? release;
        options.dist = options.dist ?? packageInfo.buildNumber;
```

I believe here we should check if there was already a value, if the user has overwritten with the `OptionsConfiguration` callback, we should honor it